appveyor.yml

```diff
@@ -1,21 +1,47 @@
 environment:
   matrix:
-  # Releases
-   - JULIA_URL: "https://julialang-s3.julialang.org/bin/winnt/x64/0.5/julia-0.5-latest-win64.exe"
-  # Nightlies
+  - JULIAVERSION: "julialang/bin/winnt/x86/0.5/julia-0.5-latest-win32.exe"
```

Comment:
this un-does https://github.com/Nemocas/Nemo.jl/pull/173, you should use the new-style urls